### ISSUE TITLE
Please improve build instructions to ultimately make builds reproducible

### ISSUE THREAD

**Issue by Giszmo** [btcontract/lnwallet#20]
As discussed [here](https://gitlab.com/walletscrutiny/walletScrutinyCom/-/issues/106), the compilation still fails with the provided build instructions and I couldn't figure out how to make it work. A detailed protocol of what I used and what I tried can be found [here](https://walletscrutiny.com/posts/2019/11/bictcoinlightningwallet/).

**Comment by nanotube** [comment]
Agreed, hopefully this can be clarified. There's related discussion of the issue here:
https://walletscrutiny.com/posts/2019/11/bictcoinlightningwallet/

**Comment by Giszmo** [comment]
More apps joined the ranks of reproducible apps. How is this wallet doing on that front?

**Comment by Giszmo** [comment]
No reply in half a year :(

**Comment by btcontract** [comment]
Sorry, project is not actively developed currently, the plan is to provide maintenance at most. 

Currently I actively work on https://github.com/btcontract/firebird which aims to be a deterministic build when ready.

**Comment by Giszmo** [comment]
So lnwallet is defunct? https://play.google.com/store/apps/details?id=com.btcontract.wallet appears to not be live yet.

**Comment by btcontract** [comment]
Firebird is not in fully functional state yet, not present on any app store currently. 

BLW is usable and there are no plans to shut it down but there is no active development either.

**Comment by btcontract** [comment]
@Giszmo 

> So lnwallet is defunct? https://play.google.com/store/apps/details?id=com.btcontract.wallet appears to not be live yet.

It's alive now and is a successor of BLW, repository is at https://github.com/btcontract/wallet. I'd be glad if you could scrutinize it, I'm ready to provide help and answer questions if needed.

**Comment by Giszmo** [comment]
Ok. Updated [the relevant analysis](https://walletscrutiny.com/android/com.btcontract.wallet/#analysis).
